scripts/tags_and_languages.py

```diff
@@ -0,0 +1,338 @@
+"""
+This Python code: 
+ - compare tags on products (using hugging face dataset) together with the 
+taxonomy
+ - for all tags being unknown but having a known element in the taxonomy for 
+ another language, update the text defining the tags
+ - finally, update the products
+
+To use a virtual environment (it depends on the OS:
+  https://python.land/virtual-environments/virtualenv)
+```python3.xx -m venv venv```
+```source venv/bin/activate```
+```pip install requests```
+```pip install duckdb```
+```pip install numpy```
+```pip install pandas```
+
+- add your username and password in the variable
+
+- run the code with:
+```python3 tags_per_languages.py```
+"""
+
+from datetime import datetime
+import duckdb
+import re
+import requests
+import sys
+import unicodedata
+
+data = {
+    'user_id': "TODO",
+    'password': "TODO",
+}
```

Comment:
call is BASE_DATA and underneath use:

data = dict(BASE_DATA, myvalue=1, othervalue="foo")